src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/haflows/HaFlowRerouteSpec.groovy

```diff
@@ -102,7 +114,9 @@ class HaFlowRerouteSpec extends HealthCheckSpecification {
         then: "The HA-flow goes to 'Down' status"
         wait(rerouteDelay + WAIT_OFFSET) {
             haFlowHelper.assertHaFlowAndSubFlowStatuses(haFlow.haFlowId, FlowState.DOWN)
-            // TODO check failed reroute in history https://github.com/telstra/open-kilda/issues/5169
+            haFlowHelper.getHistory(haFlow.haFlowId).getEntriesByType(HaFlowActionType.REROUTE_FAIL).last().payloads.find {
```

Comment:
Let's remove this check from this test because you already checked it in HaFlowHistorySpec test.

Reply:
Tagged reroute-related test as "LOW_PRIORITY" in the HaFlowHistorySpec, decided to leave checking in one positive and one negative case.